modules/dynamic-plug-in-sdk-extensions.adoc

```diff
@@ -1427,8 +1409,8 @@ events
 
 === Summary
 
-BuildAdapter contributes an adapter to adapt element to data that can be
-used by Build component
+`BuildAdapter` contributes an adapter to adapt element to data that can be
```

Comment:
I don't understand the phrase "to adapt element to data" that is here and in the following few changed lines